### PR TITLE
Windows performance hint

### PR DESCRIPTION
**Pull request by robinbrisa**
Added commented lines in _docker-compose.override.yml_ about improving performance on Windows by using Docker volumes for the vendor and var directories.

Information found in issue #242.

### CODE REVIEW

File: docker-compose.override.yml [hunk 1]
```diff
@@ -9,6 +9,11 @@ services:
       # If you develop on Mac you can remove the var/ directory from the bind-mount
       # for better performance by enabling the next line
       # - /srv/app/var
+      # If you develop on Windows you can greatly improve performance by using Docker
+      # volumes for the vendor and var directories by enabling the next 2 lines and 
+      # the "volumes" block at the bottom of this file
+      # - vendor:/srv/app/vendor
+      # - var:/srv/app/var
```
Comment (dunglas) [inline docker-compose.override.yml:16]: Couldn't we use the same syntax as for Mac? This should have the same performance benefit.

File: docker-compose.override.yml [hunk 2]
```diff
@@ -9,6 +9,11 @@ services:
       # If you develop on Mac you can remove the var/ directory from the bind-mount
       # for better performance by enabling the next line
       # - /srv/app/var
+      # If you develop on Windows you can greatly improve performance by using Docker
```
Comment (dunglas) [inline docker-compose.override.yml:12]: This works for both Mac and Windows. I suggest merging both comments.

**Review by dunglas — CHANGES_REQUESTED** [review]

**Comment by robinbrisa** [comment]
Salut Kévin, you're right, using the same syntax works indeed. 
I just made some tests and the vendor dir needs to be added as it's the one which makes the most difference performance-wise. Not sure if it's the same on Mac though. 
I made a new version to reflect this, removing the volumes and adding the vendor directory to the bind-mount exclusion.

**Review by dunglas — APPROVED** [review]

**Comment by dunglas** [comment]
Thanks! And yes excluding the vendors improves performance on Mac too, but the drawback is that it's harder to hack a dependency (when working with a private Packagist instance hosting company packages or when hacking an open source library).

**Pull request merged**